pkg/controllers/dashboard/helm/repo.go

```diff
@@ -80,6 +76,12 @@ func (r *repoHandler) ClusterRepoDownloadEnsureStatusHandler(repo *catalog.Clust
 	if registry.IsOCI(repo.Spec.URL) {
 		return status, nil
 	}
+
+	interval := 6 * time.Hour
```

Comment:
Maybe we can use the setting `SystemFeatureChartRefreshSeconds` as the default value?